lua/neotest-dart/init.lua

```diff
@@ -97,7 +97,7 @@ local function get_strategy_config(strategy, path, script_args)
       local dap_command = 'flutter'
       if command:find('fvm ') ~= nil then
         local flutter_bin_symlink =
-          utils.join_path(vim.loop.cwd(), '.fvm', 'flutter_sdk', 'bin', 'flutter')
+            utils.join_path(vim.loop.cwd(), '.fvm', 'flutter_sdk', 'bin', 'flutter')
```

Comment:
I am using `stylua` to format code, it has `indent_width = 2` setting in `stylua.toml`, this change will conflict with the predefined configuration.